util/util.go

```diff
@@ -0,0 +1,291 @@
+package util
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"os/user"
+	"path/filepath"
+
+	"github.com/lbryio/lbry.go/extras/errors"
+	"github.com/lbryio/lbry.go/lbrycrd"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/filters"
+	"github.com/docker/docker/client"
+	"github.com/mitchellh/go-ps"
+	log "github.com/sirupsen/logrus"
+)
+
+func GetBlobsDir() string {
+	blobsDir := os.Getenv("BLOBS_DIRECTORY")
+	if blobsDir == "" {
+		usr, err := user.Current()
+		if err != nil {
+			log.Error(err.Error())
+			return ""
+		}
+		blobsDir = usr.HomeDir + "/.lbrynet/blobfiles/"
+	}
+
+	return blobsDir
+}
+
+func IsBlobReflectionOff() bool {
+	return os.Getenv("REFLECT_BLOBS") == "false"
+}
+
+func GetLBRYNetDir() string {
+	lbrynetDir := os.Getenv("LBRYNET_DIR")
+	if lbrynetDir == "" {
+		usr, err := user.Current()
+		if err != nil {
+			log.Errorln(err.Error())
+			return ""
+		}
+		return usr.HomeDir + "/.lbrynet/"
+	}
+	return lbrynetDir
+}
+
+const ALL = true
+const ONLINE = false
+
+func GetLBRYNetContainer(all bool) (*types.Container, error) {
+	return getDockerContainer("lbrynet", all)
+}
+
+func getDockerContainer(name string, all bool) (*types.Container, error) {
+	cli, err := client.NewEnvClient()
+	if err != nil {
+		panic(err)
+	}
+	filters := filters.NewArgs()
+	filters.Add("name", name)
+	containers, err := cli.ContainerList(context.Background(), types.ContainerListOptions{All: all, Filters: filters})
+	if err != nil {
+		panic(err)
```

Comment:
panic?

Reply:
good catch...testing..

Reply:
actually this is a good idea. It only runs if docker is used and we want to just crash if it cant communicate with docker.